splinterd/src/main.rs

```diff
@@ -148,6 +148,8 @@ fn main() {
           "If set tls should accept all peer certificates")
         (@arg bind: --("bind") +takes_value
           "Connection endpoint for REST API")
+        (@arg database: --("db") +takes_value
```

Comment:
Here we use '--db' but in the configuration file it is 'database'. Suggest we use '--database'/database or '--db-url'/db_url so we have consistency.

Reply:
updated to --database